Game2048Player/src/test/java/manual/PlayGameManualTest.java

```diff
@@ -1,4 +1,4 @@
-package playgame;
+package manual;
```

Comment:
I'm not sure I'm a fan of this `manual` nomenclature, mostly because I think it's the default. If something isn't a bot, it's a player. But up to you.